packages/react-vapor/src/components/badge/tests/Badge.spec.tsx

```diff
@@ -1,5 +1,5 @@
+import {render, screen, within} from '@test-utils';

```

Comment:
test-utils being the new name for octopus? :octopus:

Reply:
yes pretty much